src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reverseKeys = {};
```

Comment:
```suggestion
  const reversedRobot = {};
```
Actually, it's not keys, but a new robot